savoirfairelinux/sous-chef
# Fixed #803: order to create must not be empty

## Fixes #803 by lingxiaoyang

### Changes proposed in this pull request:

* While validating the order batch creation form, for each delivery date, check if quantities and three checkboxes are all empty. If so, highlight these forms as red, and prevent the form submission.

![image](https://user-images.githubusercontent.com/8630726/29731043-da8786b2-89b0-11e7-86a9-36234a5612ca.png)

* A small JS change is needed to flush the checkbox errors on newly selected dates.

### Status


### How to verify this change

1. Merge and gulp
2. Go to order batch creation form, select client, select a date, set size, and submit the form. You should then see error indication on the quantity fields and checkboxes.
3. Try back-and-forth with more dates.

### Deployment notes and migration

none

### New translatable strings


### Additional notes

none
